src/types/Task.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Task
+ * ------------------------------------------------
+ * DESCRIPTION_HERE
```

Comment:
Let's add a description here! (or just remove this comment entirely)

Reply:
decided to delete the comment -- I feel like the type is pretty self-explanatory! It seemed messier to try to "explain it". I did rename "task" to "description" because that makes more sense, and it matches the placeholder text in the input.